examples/layer-browser/src/app.js

```diff
@@ -102,6 +102,12 @@ class App extends PureComponent {
     setImmutableDataSamples(settings.immutable);
   }
 
+  _onQueryFeatures() {
```

Comment:
Should we call it onQueryElements or onQueryObjects for consistency? We should review or documentation of the `data` prop and use the same nomenclature for the individual objects...